validate/validate.go

```diff
@@ -519,6 +520,12 @@ func (v *Validator) CheckLinux() (msgs []string) {
 		}
 	}
 
+	if v.spec.Linux.MountLabel != "" {
+		if err := label.Validate(v.spec.Linux.MountLabel); err != nil {
```

Comment:
The validation of mountLabel is not host-specific. It just checks whether label contains unexpected options. Please refer to [label.Validate()](https://github.com/opencontainers/selinux/blob/master/go-selinux/label/label_selinux.go#L189)